### PR TITLE
Added link from the troubleshooting page to blog post

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

## Which problem is this PR solving?
- Partly solves #128

### CODE REVIEW

File: content/docs/next-release/troubleshooting.md [hunk 1]
```diff
@@ -3,4 +3,6 @@ title: Troubleshooting
 description: Solve commonly encountered issues
 weight: 10
 draft: true
```
Comment (yurishkuro) [inline content/docs/next-release/troubleshooting.md]: Remove draft?

**Comment by yurishkuro** [comment]
Maybe add to 1.9 docs as well

File: content/docs/next-release/troubleshooting.md (hunk 1 above)
Comment (jpkrohling) [reply content/docs/next-release/troubleshooting.md]: I removed, but doesn't seem to have an effect?

Comment (yurishkuro) [reply content/docs/next-release/troubleshooting.md]: how do you mean? It shows up under 1.9 now

image

**Pull request merged**